InterfaceParser/model/enum_subset.py

```diff
@@ -0,0 +1,38 @@
+"""Enumeration subset.
+
+"""
+
+from model.interface_item_base import InterfaceItemBase
+
+
+class EnumSubset(InterfaceItemBase):
```

Comment:
I am not familiar with enum subset. Is this what the resultCode enum uses?

Reply:
enum_subset used for instantiating each enum "element" from xml and then butting as attribute "elements" into enum.py instance.